wp-content/themes/csisjti/assets/_scss/components/_facets.scss

```diff
@@ -202,6 +202,10 @@
     text-overflow: ellipsis;
     word-break: break-word;
 
+    @media screen and (min-width: 64em) and (max-height: 1000px) {
```

Comment:
I would  make this the same media query that we're using for resizing the height, otherwise they don't change in sync with each other.